### ISSUE TITLE
PagerTitleIndicator does not scroll when it has many options.

### ISSUE THREAD

**Issue by Drey23** [zbtang/React-Native-ViewPager#58]
Hi,

When I add many options to the PagerTitleIndicator, this one does not fit and it does not scroll the options.

**Comment by zbtang** [comment]
Yes, PagerTitleIndicator not support scroll yet . But you can implement your Indicator.

**Comment by tyforcode** [comment]
I was able to get it overflowing titles scrolling by adding trackScroll={true} to the PagerTitleIndicator.

It's in the code but not the documentation
